### PR TITLE
feat: Litmus 3.0: Added enhancements in graphql server

### PR DESCRIPTION
**Pull request by Saranya-jena**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: chaoscenter/graphql/definitions/shared/chaos_infrastructure.graphqls [hunk 1]
```diff
@@ -0,0 +1,678 @@
...
+  """
+  infraNsExists: Boolean
+  """
+  Bool value indicating whether service account used already exists on infra or not
+  """
+  infraSaExists: Boolean
+  """
+  Timestamp of the last experiment run in the infra
+  """
+  lastWorkflowTimestamp: String
```
Comment (SarthakJain26) [inline chaoscenter/graphql/definitions/shared/chaos_infrastructure.graphqls:97]: lastExperimentTimestamp* ^^

**Review by SarthakJain26 — CHANGES_REQUESTED** [review]

**Review by imrajdas — APPROVED** [review]

**Review by vanshBhatia-A4k9 — APPROVED** [review]
3.0 🚀

File: chaoscenter/graphql/server/graph/chaos_infrastructure.resolvers.go [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+	return r.chaosInfrastructureService.ConfirmInfraRegistration(request, *data_store.Store)
+}
+
+func (r *mutationResolver) DeleteInfras(ctx context.Context, projectID string, infraID string) (string, error) {
+	logFields := logrus.Fields{
+		"projectId":    projectID,
+		"chaosInfraId": infraID,
+	}
+
+	logrus.WithFields(logFields).Info("request received to delete chaos infrastructures")
```
Comment (vanshBhatia-A4k9) [inline chaoscenter/graphql/server/graph/chaos_infrastructure.resolvers.go:54]:
```suggestion
func (r *mutationResolver) DeleteInfra(ctx context.Context, projectID string, infraID string) (string, error) {
	logFields := logrus.Fields{
		"projectId":    projectID,
		"chaosInfraId": infraID,
	}

	logrus.WithFields(logFields).Info("request received to delete chaos infrastructure")
```

**Review by SarthakJain26 — APPROVED** [review]
LGTM

**Pull request merged**